cypress/support/step-definitions/switch-steps.js

```diff
@@ -40,3 +40,11 @@ Then('Switch component is not loading', () => {
   switchInput().should('not.have.attr', 'disabled');
   switchInput().should('not.have.attr', 'readonly');
 });
+
+Then('I click onto {int} switch', (position) => {
+  switchDataComponent(position).click();
+});
+
+Then('I click onto {int} switch again', (position) => {
```

Comment:
we don't need to copy/paste same code, just make it like -> 
`When('I hit Tab key {int} time(s)', (times) => {
  pressTABKey(times);
});`

Reply:
fixed